coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -206,7 +207,10 @@ public void accessCluster(AccessClusterRequest request, StreamObserver<AccessClu
     AccessClusterResponse response;
     AccessManager accessManager = coordinatorServer.getAccessManager();
 
-    AccessInfo accessInfo = new AccessInfo(request.getAccessId(), Sets.newHashSet(request.getTagsList()));
+    Map<String, String> extraProperties = request.getExtraPropertiesMap();
+    AccessInfo accessInfo =
+            new AccessInfo(request.getAccessId(), Sets.newHashSet(request.getTagsList()),
+                    extraProperties == null ? Collections.emptyMap() : extraProperties);
```

Comment:
It'd be better to let the `AccessInfo` handle the case where the parameter `extraProperties` is null in its constructor.

Reply:
done